### PR TITLE
Dominator colors are now directly determined by the gang's color

### LINKED ISSUE #27603: White gang dominators are invisible
@Robustin dominators aren't actually colored sprites they're specific icon states

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #27603
Looks 200% better

### CODE REVIEW

File: code/game/gamemodes/gang/dominator.dm [hunk 1]
```diff
@@ -196,9 +211,9 @@
 		priority_announce("Network breach detected in [locname]. The [gang.name] Gang is attempting to seize control of the station!","Network Alert")
 		gang.domination()
 		SSshuttle.registerHostileEnvironment(src)
-		src.name = "[gang.name] Gang [src.name]"
+		name = "[gang.name] Gang [name]"
 		operating = 1
-		icon_state = "dominator-[gang.color]"
+		update_icon()
 
 		countdown.color = gang.color_hex
```
Comment (lzimann) [inline code/game/gamemodes/gang/dominator.dm]: Won't this countdown look really bad having the same color as the dom?

File: code/game/gamemodes/gang/dominator.dm [hunk 2]
```diff
@@ -196,9 +211,9 @@
 		priority_announce("Network breach detected in [locname]. The [gang.name] Gang is attempting to seize control of the station!","Network Alert")
 		gang.domination()
 		SSshuttle.registerHostileEnvironment(src)
-		src.name = "[gang.name] Gang [src.name]"
+		name = "[gang.name] Gang [name]"
```
Comment (Cyberboss) [inline code/game/gamemodes/gang/dominator.dm]: src instead of name

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/gang/dominator.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/gamemodes/gang/dominator.dm]: Tunnel Gang the dominator, sure

File: code/game/gamemodes/gang/dominator.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/gang/dominator.dm]: It already does sooooo I don't think so no

Comment (MrPerson) [inline code/game/gamemodes/gang/dominator.dm]: Changing it to not colorize the text would be trivial if someone wants to do that

**Pull request merged**